package.json

```diff
@@ -67,12 +65,20 @@
     "vue-template-compiler": "^2.x"
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/plugin-proposal-decorators": "^7.3.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-typescript": "^7.1.0",
+    "@babel/runtime-corejs2": "^7.3.1",
     "@vue/component-compiler-utils": "^2.4.0",
+    "babel-jest": "24.1.0",
```

Comment:
@yohodopo / @eddyerburgh Is there a reason why we pin this to `24.1.0`? Should this maybe be a peer dependency and/or also point to a version range with `^`?